### PR TITLE
QOLOE-135 card, accordion, type visual adjustments

### PR DESCRIPTION
**Pull request by re-becca**
See details at [https://ssq-qol.atlassian.net/browse/QOLOE-135](https://ssq-qol.atlassian.net/browse/QOLOE-135)

### CODE REVIEW

File: src/scss/qld-type.scss [hunk 1]
```diff
@@ -109,26 +115,22 @@ strong, b, th {
...
 a.nav-link {
-  text-decoration: underline;
-  text-underline-offset: var(--qld-link-underline-offset) !important;
-  text-decoration-thickness: var(--qld-link-underline-thickness) !important;
+  text-underline-offset: var(--qld-link-underline-offset);
+  text-decoration-thickness: var(--qld-link-underline-thickness) !important;  // Overrides default BS styles
 
   &:hover {
-    text-decoration-thickness: var(--qld-link-underline-thickness-hover) !important;
+    text-decoration-thickness: var(--qld-link-underline-thickness-hover) !important; // Overrides default BS styles
```
Comment (elvishu) [inline src/scss/qld-type.scss:125]: I'm curious if we can override BS5 without an !important tag.
Totally understand if timing is tight atm.

**Review by elvishu — DISMISSED** [review]

File: src/scss/qld-type.scss (hunk 1 above)
Comment (re-becca) [reply src/scss/qld-type.scss:125]: I wasn't able to find where, and wasn't sure why this didn't work as an override without the !important. Keen if anyone can shed some light.

Comment (stvp-qld) [reply src/scss/qld-type.scss:125]: In theory we should be able to rescope a bootstrap CSS variable...like --text-decoration-thickness or similar.  But  Bootstrap might not have a variable setup for that yet... I know it's ongoing part of their project to swap out hard coded values with variables but still a few gaps.

Failing that, just the fact that our CSS is rendered _after_ the bootstrap CSS should mean our CSS is used by the browser.   Bootstrap deliberatly removed all of their !important tags for this reason, so when a rule is redefined further along in the CSS, that later rule is honoured.

Comment (stvp-qld) [reply src/scss/qld-type.scss:125]: You might have been competing with this one @re-becca?  Was it inside sidenav?

image

Comment (re-becca) [reply src/scss/qld-type.scss:125]: @stvp-qld It's coming from Bootstrap's reboot.scss. 'text-decoration: underline' being defined there somehow then prevents 'text-decoration-thickness' being applied without the !important
Screenshot 2024-06-20 at 1 08 03 pm
This is across any component that inherits 'a' styles - cards, typography.

Comment (re-becca) [reply src/scss/qld-type.scss:125]: Ah looks like 'reboot.scss' is imported after 'qld-type.scss' unlike components.scss, will chat to you @stvp-qld  about how best to handle this

**Review by elvishu — APPROVED** [review]
Awesome work, Bec.

**Comment by re-becca** [comment]
Noticed one more heading colour issue to fix, before merging. BBS.

**Review by ghazal-salehi — APPROVED** [review]

**Pull request closed without merging**